docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -567,30 +589,30 @@ Let's suppose we have a template called `item.html`:
 <1> `showImage` is a named parameter.
 <2> `it` is a special key that is replaced with the first unnamed param of the tag.
 
-Now if we register this template under the name `item` and if we add a `UserTagSectionHelper` to the engine:
+Now if we register this template under the name `itemDetail` and if we add a `UserTagSectionHelper` to the engine:
 
 [source,java]
 ----
 Engine engine = Engine.builder()
-                   .addSectionHelper(new UserTagSectionHelper.Factory("item"))
+                   .addSectionHelper(new UserTagSectionHelper.Factory("myItem","itemDetail.html"))
                    .build();
 ----
 
-NOTE: In Quarkus, all files from the `src/main/resources/templates/tags` are registered and monitored automatically.
+NOTE: In Quarkus, all files from the `src/main/resources/templates/tags` are registered and monitored automatically!
 
 We can include the tag like this:
 
 [source,html]
 ----
 <ul>
-{#each items}
+{#for item in items}
   <li>
-  {#item this showImage=true /} <1>
+  {#myItem item showImage=true /} <1>
```

Comment:
Huh, why did you rename it? Does the `#item` tag namespace clash with the local variable `item`? If that's the case it's a bug.

Reply:
No, it could be `item` or anything else. I just wanted to have the example more readable... `{#item item}` is not good.

Reply:
OK, pfew :)

But frankly `#{item item}` is still my better than `#{myItem item}`. Perhaps it's `#{itemDescription item}` you were aiming for? ;)

Reply:
> But frankly `#{item item}` is still way better than `#{myItem item}`.

Oh, really? ;-) Anyway, `#{itemDescription item}` or `{#itemDetail item}` reads definitely better. My intention was to show the link between the tag name registered above. But this is not important in quarkus where tags are registered automatically.